libakumuli/storage_engine/blockstore.h

```diff
@@ -117,37 +117,72 @@ class FixedSizeFileStorage : public BlockStore,
     std::vector<std::string> volume_names_;
 
     //! Secret c-tor.
-    FixedSizeFileStorage(std::string metapath, std::vector<std::string> volpaths);
+    FileStorage(std::string metapath, std::vector<std::string> volpaths);
 
+    virtual void update_current_volume() = 0;
```

Comment:
That's better than the current names for sure - I didn't think of anything better. I'll update to use those.